client/src/components/Dashboard/Header/Authentication/Signup.jsx

```diff
@@ -1,80 +1,34 @@
-import React, { Component } from 'react';
+import React from 'react';
 import {
   Form, Button, Container, Row, Col,
 } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import { Redirect } from 'react-router-dom';
-import { Toaster, Intent } from '@blueprintjs/core';
-import db from '@src/fire';
+import * as loginActions from '../../../../actions/loginationActions';
```

Comment:
You can use path alias here also:

import * as loginActions from '@src/actions/loginationActions';